### PR TITLE
redoing transaction updates for serverless in a new branch

### PR DESCRIPTION
**Pull request by ajjindal**
Support for Performance on Python/Servelress

### CODE REVIEW

**Review by untitaker — APPROVED** [review]

File: src/platforms/python/guides/aws-lambda/index.mdx [hunk 1]
```diff
@@ -52,20 +59,13 @@ sentry_sdk.init(
...
-<!-- TODO-ADD-VERIFICATION-EXAMPLE -->
-
 ## Behavior
 
 With the AWS Lambda integration enabled, the Python SDK will:
 
-- Automatically report all exceptions from your Lambda Functions
-- Issues reports automatically include:
+- Automatically report all events from your Lambda Functions
+- You can control transactions sample rate using `traces_sample_rate`. You can learn more about transactions in <PlatformLink to="/configuration/filtering/">Distributed Tracing</PlatformLink>
```
Comment (PeloWriter) [inline src/platforms/python/guides/aws-lambda/index.mdx:67]:
```suggestion
- Automatically report all events from your Lambda Functions. You can [modify the transaction sample rate] <PlatformLink to="/configuration/filtering/">Distributed Tracing</PlatformLink> using `traces_sample_rate`.
```

File: src/platforms/python/guides/gcp-functions/index.mdx [hunk 2]
```diff
@@ -60,7 +62,8 @@ The timeout warning is sent only if the timeout in the GCP Function configuratio
 
 With the GCP Functions integration enabled, the Python SDK will:
 
-- Automatically report all exceptions from your google cloud functions
+- Automatically report all events from your Lambda Functions
+- You can control transactions sample rate using `traces_sample_rate`. You can learn more about transactions in <PlatformLink to="/configuration/filtering/">Distributed Tracing</PlatformLink>
```
Comment (PeloWriter) [inline src/platforms/python/guides/gcp-functions/index.mdx:66]:
```suggestion
- Automatically report all events from your Lambda Functions. You can [modify the transaction sample rate] <PlatformLink to="/configuration/filtering/">Distributed Tracing</PlatformLink> using `traces_sample_rate`.
```

**Review by PeloWriter — APPROVED** [review]
minor edits. Otherwise, LGTM

File: src/platforms/python/guides/gcp-functions/index.mdx (hunk 2 above)
Comment (untitaker) [reply src/platforms/python/guides/gcp-functions/index.mdx:66]: The rephrasing of point 2 looks good but the first and second bulletpoint are not related to each other in any way.

**Pull request merged**